### ISSUE TITLE
ITMS-90809: Deprecated API Usage - use WKWebView instead of UIWebView

### ISSUE THREAD

**Issue by Samy-Nagy** [Tealium/tealium-ios#130]
apple sending warning message about using deprecated API Usage UIWebView, after scanning my project folder found someThing related to Tealium
pod 'TealiumIOS', '5.6.6'
pod 'TealiumIOSLifecycle', '5.6.6'

grep -r UIWebView .
Binary file ./Pods/TealiumIOSLifecycle/TealiumIOSLifecycle.framework.dSYM/Contents/Resources/DWARF/TealiumIOSLifecycle matches
Binary file ./Pods/TealiumIOS/TealiumIOS.framework.dSYM/Contents/Resources/DWARF/TealiumIOS matches

**Comment by craigrouse** [comment]
@SamiYousef It appears our dSYMs may be out of date, which is why you're seeing this. Rest assured there is no UIWebView code in the framework itself. We'll address this and generate some fresh dSYM files to stop Apple picking this up.

**Comment by craigrouse** [comment]
@SamiYousef I should mention also, it may be preferable for you to use our newer Swift library: https://github.com/tealium/tealium-swift.

**Comment by Samy-Nagy** [comment]
ok thanks @craigrouse

**Comment by RGG-jayoung** [comment]
@craigrouse are you sure there is no UIWebView left overs in the framework? Downloading the latest release I see code in `WebViewController.swift` and `AAPLWebViewController.m/h`.

<img width="1598" alt="image" src="https://user-images.githubusercontent.com/53495188/90823642-36fac580-e304-11ea-83ba-b3feaea3ca22.png">

<img width="506" alt="image" src="https://user-images.githubusercontent.com/53495188/90823857-86d98c80-e304-11ea-8c6a-fc121d034584.png">

**Comment by craigrouse** [comment]
@RGG-jayoung those are in the sample app and do not form part of the binary.

**Comment by RGG-jayoung** [comment]
Thanks @craigrouse. I understand that these are part of the sample, but this still matches on the dSYM and we still are getting warnings from apple on submission. We've verified all other dependencies and this is the only hold out `tealium-ios-master/TealiumIOS.framework.dSYM/Contents/Resources/DWARF/TealiumIOS`

**Comment by craigrouse** [comment]
@RGG-jayoung OK - leave it with us. Despite there being no reference to it in the source project, it still shows up in a grep on the dSYMs for the latest release, so they must not have generated properly. We'll try and resolve this ASAP.

**Comment by jonathanswong** [comment]
@RGG-jayoung 
**dSYMs** do not contain any functional code, just debugging metadata. Because we support iOS 9.2, this version may contain `UIWebView` (even though our library does not use it). Instead, please try

`nm TealiumIOS.framework/TealiumIOS | grep -i UIWebView`

This should return no results. You may want to try this on other dependencies. Please let us know if you are still having issues. Thanks.

**Comment by jonathanswong** [comment]
Hi @RGG-jayoung , did this solve your issues? Thanks.

**Comment by jonathanswong** [comment]
@RGG-jayoung Closing this issue for now. Please let us know if you are still having this issue. Thanks.

**Comment by RGG-jayoung** [comment]
Thank you both for the help. We moved our implementation over to the Swift SDK which resolved the issue. To confirm things we did some tests, with the 5.6.6+ submitting to the App Store still fired the UIWebView warning (maybe the dSYMs?). Unclear what Apple is searching for under the hood but the warning fired nonetheless. When submitting with the Swift library we no longer see the warning.

Thanks again!

@jonathanswong P.S. great course on Pluralsight with Xcode, I promote it heavily with my team :)
